src/features/Students/StudentsPage/index.jsx

```diff
@@ -35,6 +38,7 @@ const StudentsPage = () => {
     <Container size="xl" className="px-4">
       <h2 className="title-page">Students</h2>
       <div className="page-content-container">
+        <StudentsFilters />
```

Comment:
I still see the error, in the pagination I lost the filters
![filters](https://github.com/user-attachments/assets/3a456aaa-3967-4a9b-9154-547d111bf7e9)

You have to add the filters in this page in the fetchStudentsData